src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -55,6 +56,7 @@ interface Props {
   className: string;
   indexPatterns: IIndexPattern[];
   intl: InjectedIntl;
+  fixFilterBar?: boolean;
```

Comment:
nit: Can you just add a comment for what this "fixes"?

Reply:
updated to use a new naming `isFilterBarPortable` that self explains, and also updated the related logic